ibis/omniscidb/tests/test_client.py

```diff
@@ -100,6 +100,53 @@ def test_union_op(alltypes):
         expr.compile()
 
 
+def test_add_column(con):
+    table_name = 'my_table'
```

Comment:
could you add test  for:

- 0 field
- 1 field (done)
- 2 or more fields (with different types)

thanks!

Reply:
Done

Reply:
@YarShev thanks for adding more parameters for the tests here!

I think you could to refactor a little bit your code here. you can try something like this:

```
@pytest.mark.parametrize(
    'cols_with_types',
    [
        {},
        {'c': 'double'},
        {'c': 'double', 'd': 'string', 'e': 'point', 'f': 'polygon', 'g': 'int16'},
    ],
)
def test_add_column(con, new_cols):
    table_name = 'my_table'
    con.drop_table(table_name, force=True)

    schema = ibis.schema([('a', 'float'), ('b', 'int8')])

    con.create_table(table_name, schema=schema)

    if len(new_cols) == 0:
        with pytest.raises(IbisInputError):
            con.add_column(table_name, cols_with_types)
        return con.drop_table(table_name)

    con.add_column(table_name, new_cols)

    schema_new_cols = ibis.schema(new_cols.items())
    schema_new = schema.append(schema_new_cols)

    try:
        t = con.table(table_name)
        assert schema == schema_new
    finally:
        con.drop_table(table_name)
```

note: probably for ddl we should use the same type names used by ibis expressions .. not the ones directly used by omniscidb. it could be a little bit confusing . I have a WIP PR that I was adding the add_columns op too. As you are working on that I will just remove that from my code ... but for now you can take a look how I was treating the datatypes there: https://github.com/ibis-project/ibis/pull/2086